soul-plugin/soul-plugin-divide/src/main/java/org/dromara/soul/plugin/divide/DividePlugin.java

```diff
@@ -51,11 +52,22 @@
 @Slf4j
 public class DividePlugin extends AbstractSoulPlugin {
 
+    @SneakyThrows
     @Override
     protected Mono<Void> doExecute(final ServerWebExchange exchange, final SoulPluginChain chain, final SelectorData selector, final RuleData rule) {
         final SoulContext soulContext = exchange.getAttribute(Constants.CONTEXT);
         assert soulContext != null;
         final DivideRuleHandle ruleHandle = UpstreamCacheManager.getInstance().obtainHandle(DividePluginDataHandler.getCacheKeyName(rule));
+        if (exchange.getRequest().getHeaders().values().toString().getBytes("utf-8").length > ruleHandle.getHeaderMaxSize()) {
```

Comment:
this is foreach values ， then add

Reply:
get it

Reply:
for (Map.Entry<String, List<String>> headerEntry : headers.entrySet()) {
	
					for (String value :  headerEntry.getValue()) {
						 (long) value.getBytes().length;
					}
				}